### DISCUSSION TITLE
Hide a discussion

### DISCUSSION THREAD

**Discussion by trevthom** [stackernews/stacker.news#3010] [Ideas]
(Just realized the 'Issues' section has the ability for feature request so ignore the one in 'Discussions')

I would love the ability to hide a discussion that doesn't interest me at all. Sometimes I will click a discussion that I want to look at later and decide if I want to zap it, but sorting through other discussions makes it difficult to find (and I don't want to bookmark it).

**Comment by trevthom** [comment]
Much like Reddit, it would also be nice to have a feature to hide a discussion once it's been zapped.

**Comment by huumn** [comment]
Thanks for the request! We've had similar requests in the past. Something like "archive" in an email inbox.
